api_tests/nodes/views/test_node_forks_list.py

```diff
@@ -191,6 +191,20 @@ def test_forks_list_does_not_show_registrations_of_forks(
         # confirm registration of fork does not show up in public data
         assert len(res.json['data']) == 0
 
+    def test_forks_list_does_not_show_deleted_forks(
```

Comment:
Could you combine this with the above `test_forks_list_does_not_show_registrations_of_forks` to make `test_forks_list_does_not_show_registrations_of_nor_deleted_forks`? Setup for these takes forever, so we try to avoid when possible.